<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tabView">
     <attribute name="title">
      <string>View</string>
     </attribute>
     <widget class="QGroupBox" name="groupView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>271</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>What would you like to view?</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>191</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="View">
        <item>
         <widget class="QRadioButton" name="radioViewTenant">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Tenant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioViewRental">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Rental Details</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioViewProperty">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string> Property</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioViewFinances">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Finances</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushView">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>View</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabViewAddNew">
     <attribute name="title">
      <string>Add New</string>
     </attribute>
     <widget class="QGroupBox" name="groupAddNew">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>What would you like to add?</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>161</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="addNew">
        <item>
         <widget class="QRadioButton" name="radioAddNewTenant">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Tenant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioAddNewRental">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Rental Contract</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioAddNewProperty">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string> Property</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioAddNewExpense">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string> Expense</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioAddNewIncome">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Income</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushAdd">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabUpdate">
     <attribute name="title">
      <string>Update</string>
     </attribute>
     <widget class="QGroupBox" name="groupAddNew_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>261</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>What would you like to update?</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>161</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="addNew_2">
        <item>
         <widget class="QRadioButton" name="radioUpdateTenant">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Tenant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioUpdateRental">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Rental Contract</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioUpdateProperty">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string> Property</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioUpdateExpense">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string> Expense</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioUpdateIncome">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Income</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushUpdate">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="RenewContract">
     <attribute name="title">
      <string>Renew Contract</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>540</y>
        <width>481</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_Renew">
       <item row="4" column="3">
        <layout class="QVBoxLayout" name="verticalLayout_CurrentTenant_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QRadioButton" name="radioCurrentTenant_Update">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioNotCurrent_Update">
           <property name="text">
            <string>N</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_occupancyCode_Update"/>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_tenantPhone_Update">
         <property name="text">
          <string>Tenant Phone Number:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="lineEdit_tenantPhone_Update"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_monthlyRent_Update">
         <property name="text">
          <string>Monthly Rent Charge</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_occupancyCode_Update">
         <property name="text">
          <string>Override Occupancy Code:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_tenantEmail_Update"/>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="spinBoxRent_Update">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="singleStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="labelCurrentTenant_Update">
         <property name="text">
          <string>Current Tenant (Y/N):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_tenantEmail_Update">
         <property name="text">
          <string>Tenant Email:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>510</y>
        <width>501</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="2">
        <layout class="QFormLayout" name="formLayoutRentalStartAndEnd_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelRentalStart_Update">
           <property name="text">
            <string>Contract Start Date:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelRentalEnd_Update">
           <property name="text">
            <string>Contract End Date:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="dateEnd_Update">
           <property name="displayFormat">
            <string>MMM/dd/yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="dateStart_Update">
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2000</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="displayFormat">
            <string>MMM/dd/yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>2</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_sewageBill__Update">
           <property name="title">
            <string>Sewage Billed to:</string>
           </property>
           <widget class="QWidget" name="layoutWidget_9">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>58</width>
              <height>42</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_sewageBill_3">
             <item>
              <widget class="QRadioButton" name="radioSewageTenant_Update">
               <property name="text">
                <string>Tenant</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioSewageOwner_Update">
               <property name="text">
                <string>Owner</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_Utilities__Update">
           <property name="title">
            <string>Utilities Billed to:</string>
           </property>
           <widget class="QWidget" name="layoutWidget_10">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>58</width>
              <height>42</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_Utilities_3">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item>
              <widget class="QRadioButton" name="radioUtilitiesTenant_Update">
               <property name="text">
                <string>Tenant</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioUtilitiesOwner_Update">
               <property name="text">
                <string>Owner</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>36</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>490</y>
        <width>481</width>
        <height>36</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_notes_Update">
       <item>
        <widget class="QLabel" name="label_notes__Update">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_notesTenant_Update"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelUpdate2">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>680</y>
        <width>375</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Search by inputting information into A or B :</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_SearchP">
           <property name="text">
            <string>Select Property:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_tenantFirstName_Renew">
           <property name="text">
            <string>Tenant First Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_tenantLN_Renew">
           <property name="text">
            <string>Tenant Last Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboProperty"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelfirst">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labellast">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushRenew">
         <property name="text">
          <string>Renew</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>gridLayoutWidget_4</zorder>
     <zorder>layoutWidget_6</zorder>
     <zorder>labelUpdate2</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>gridLayoutWidget_3</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>345</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
